projects/backend/controllers/render.ts

```diff
@@ -129,17 +129,35 @@ const getTags = (kicker: string, tags: Tag[]): Tag[] => {
     return [seriesTag, ...tags]
 }
 
+const listTags = ['<ul>', '<li>', '<a>']
+
+const containsListTags = (str: string): boolean =>
+    listTags.some(tag => str.includes(tag))
+
+const filterStandfirst = (standfirst?: string): string | undefined => {
+    if (standfirst) {
+        return !containsListTags(standfirst) ? standfirst : undefined
+    }
+    return undefined
```

Comment:
We can probably make this function a one liner:
```suggestion
    return standfirst && !containsListTags(standfirst) ? standfirst : undefined
```